luaui/Widgets/cmd_bar_hotkeys.lua

```diff
@@ -614,7 +614,7 @@ local function makeBindsTable(keyLayout)
         table.insert(_binds, { 'Ctrl+'..i       , "group", "set "..i        })
         table.insert(_binds, { 'Shift+'..i      , "group", "selectadd "..i })
         table.insert(_binds, { 'Ctrl+Shift+'..i , "group", "add "..i        })
-        table.insert(_binds, { 'Alt+'..i        , "group", "selecttoggle "..i     })
+        table.insert(_binds, { 'Alt+Shift+'..i        , "group", "selecttoggle "..i     })
```

Comment:
Can you replace `Ctrl+Alt` with `Alt+Shift` for the windows users?

Personally I prefer `Ctrl+Shift` but we can't have default keybindings specific for minority operating systems.

Reply:
But Ctrl+Alt isn't there

Reply:
Done